index.bs

```diff
@@ -11236,6 +11296,20 @@ running the algorithm for an {{AudioNode}}, using an <a>input
 buffer</a> and the value(s) of the {{AudioParam}}(s) of this
 {{AudioNode}} as the input for this algorithm.
 
+<h3 id="unloading-a-document">Unloading a document</h3>
+	Additional <a href=
+	"https://html.spec.whatwg.org/#unloading-document-cleanup-steps">unloading
+	document cleanup steps</a> are defined for document that uses
+	{{BaseAudioContext}}:
+
+1. Reject all the promises of {{BaseAudioContext/[[pending promises]]}} with
+	<code>InvalidStateError</code>, for each {{AudioContext}} and
+	{{OfflineAudioContext}} whose relevant global object is the same as
+	the document's associated Window.
+2. Stop all {{decoding thread}}s.
+3. <a>Queue a control message</a> to close the
```

Comment:
"close" should link to the appropriate close algorithm?  Now it's a bit unclear on what "close" means.

Reply:
It's explained in a section in `close`:

>		Running a <a>control message</a> to close an
>		{{AudioContext}} means running these steps on the
>		<a>rendering thread</a>:
>              ...

and the same for `OfflineAudioContext`.